BaseFilter/sections/specific.txt

```diff
@@ -275,6 +275,8 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+tampermonkey.net##td[style^=vertical].werl
```

Comment:
I don't see that with VPN, but it can be found in the code and looks like the leftover.